R/drive_upload.R

```diff
@@ -66,35 +66,49 @@ drive_upload <- function(file = NULL,
   path <- path %||% root_folder()
   path <- as_dribble(path)
   confirm_single_file(path)
+  existing <- NULL
```

Comment:
It doesn't feel correct that we've got `existing` and `up_id` floating around. I suspect something can still be simplified.